ui/tests/integration/components/icon-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | ic-on', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`{{ic-on}}`);
```

Comment:
where is `ic-on` coming from? I haven't seen it called this anywhere.

Reply:
Whoops, ember generators still don't like single-word components, so this was the generated test for icon.js - I'll update all of this before finalizing the PR!